paper.tex

```diff
@@ -30,59 +30,68 @@
 \newacronym{UOX}{UOX}{uranium oxide fuel}
 \newacronym{MOX}{MOX}{mixed oxide fuel}
 \newacronym{LWR}{LWR}{light water reactor}
-\newacronym{FBR}{FBR}{fast breeder reactor}
+\newacronym{SFR}{SFR}{fast breeder reactor}
 
 \begin{document}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Introduction}
 
 Fuel cycle simulations, as with any simulation process, do not produce results
 without errors.  Those errors have different sources: data uncertainties (from
-the simulator's reliance on previously estimated/measured metrics), modeling
+%% I'm not a fan of the use of the word ``errors'' here.  We should be careful
+%% since (in English) this can meam ``mistakes''.  Uncertainty is a better
+%% word, and then we'll have to distinguish between input uncertainties and
+%% output uncertainties.
+the simulator's reliance on previously estimated/measured data), modeling
 uncertainties (from simplifications made by the simulator) and the
-``operational`` uncertainty (from the operation parameter uncertainties of
-different facilities).
+``operational'' uncertainty (from the expected behavior of the facilities
+being simulated).
+%% is this operational uncertainty really just a different kind of data?
 
-Simulation errors are important in the context of treaty verification. 
+Simulation errors could be important in the context of treaty verification. 
 Some scenarios require estimating historical fissile
 material production based on records of facility operation and material
 transfer.  If the measurements of facility operation and material transfer are
 subject to uncertainty, the total production quantities will then also be
-uncertain, creating an opportunity for undetected diversion.  This study seeks
+uncertain.  Any uncertainty in fissile production quantities poses a material
+accounting challenge, possibly
+creating an opportunity for undetected diversion.  This study seeks
 to understand which measures of facility operation have the most impact on the
 uncertainty of fissile material production across a complete fuel cycle.
 To accomplish this, a Total Monte Carlo methodology will be applied to the Cyclus
 fuel cycle simulator \cite{cyclus} to estimate the effect of individual facility uncertainties on the output metrics.
+%% should there be a reference for Total Monte Carlo?
```

Comment:
I am not sure we should use the term Total Monte Carlo as it usually refers to nuclear data uncertainty propagation, maybe "statistical uncertainty propagation" is better ?